Audacia.Log.AspNetCore/LogFilterAttribute.cs

```diff
@@ -18,12 +18,12 @@ public sealed class LogFilterAttribute : Attribute, IFilterMetadata
     /// <summary>
     /// Gets or sets the names of arguments to exclude from the logs.
     /// </summary>
-    public string[] ExcludeArguments { get; set; } = Array.Empty<string>();
+    public string[] ExcludeArguments { get; set; } = [];
```

Comment:
should this attribute now be removed in leiu of `LogRequestBodyActionFilterAttribute`?